app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -123,6 +129,24 @@ def process_collection(collection, key)
         end
       end
 
+      def gather_data_for_this_region(arm_service)
+        results = []
+        @data[:resource_groups].each do |resource_group|
+          results << arm_service.list(resource_group[:name])
```

Comment:
Well, I lied. You can do this:

    res = Azure::Armrest::Resource.new(conf)
    res.list(:location => your_region).each{ |r| p r }

Keep in mind it doesn't contain a lot of detailed information if you do it that way. You'll get name, type, location and id. I don't know if that's enough.